src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke.util;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
```

Comment:
To also add on to @sk2001git, it would be better to arrange the import statements in alphabetical order. This adheres to the checkstyle. So lines 8 to 13 would be:
```ruby
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

import java.util.ArrayList;
```